tests/test.py

```diff
@@ -248,11 +249,14 @@ def test_call_chain_escaped_line_break_1():
 
 # BAD (don't use parentheses when already using \ for line continuation)
 def test_call_chain_escaped_line_break_2():
-    s = """(   \\
+    s = """( \\
     foo   \\
     )  .  bar   (   baz   )
     """
-    assert len(_results(s)) == 1
+    if "3.10" in sys.version:
```

Comment:
```suggestion
    if sys.version_info >= (3, 10):
```

Checking for a sub-string in `sys.version` is not very reliable. Here is an example version string.
`'3.9.12 (main, Jul 20 2022, 13:19:52) \n[GCC 9.4.0]'`. What if GCC would be version 3.10.5, for instance?

Plus, the suggested change will also treat any future Python version like 3.10. I assume this change in what syntax is considered valid and what isn't won't be changed back again any time soon.